<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1848</width>
    <height>1213</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>1741</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;background-color: #E0E0E0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;background-color: #E0E0E0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="ctroller" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>820</y>
      <width>851</width>
      <height>351</height>
     </rect>
    </property>
    <widget class="QPushButton" name="import_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_model">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>加载模型</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generate">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>场景图生成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="detect">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目标检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>260</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>海上场景图生成与行为理解应用验证系统</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QWidget" name="video_ctr" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>730</y>
      <width>891</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QSlider" name="progress">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>851</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="suspend">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>⏸</string>
     </property>
    </widget>
    <widget class="QPushButton" name="play">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>▶</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>770</y>
      <width>851</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListView" name="listView_obj">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView_rel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="obj_label">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>730</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目标信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="rel_label">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>730</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关系信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
